scripts/generate_sample_functions.py

```diff
@@ -110,15 +98,14 @@ def write(data, depth=0):
                 output.extend(hanging_indent(call_prefix, arg_groups, ',', ');', depth=2))
 
         write('} else {', depth=1)
+        write('DEBUG_ASSERT(kMaxChannels <= %(i)d);' % {'i': i}, depth=2)
```

Comment:
Please remove -- it's not necessary since this loop can handle an arbitrary number of channels.